### PR TITLE
Add data loader steps for Hub

### PR DESCRIPTION
**Pull request by strickvl**
Added:

- [x] Torchvision
- [x] Tensorflow 
- [x] Sklearn data importer steps (including `openml` loader)
- [ ] Huggingface hub loaders
- [ ] Langchain hub loaders
- [ ] Llama_index hub loaders

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,17 +15,20 @@ ZenML projects using `zenml init --template`
     - Digit: `sklearn_digits_loader_step`
     - Iris: `sklearn_iris_loader_step`
     - Diabetes: `sklearn_diabetes_loader_step`
-    - Breast Cancer Wisconsin: `sklearn_breast_cancer_loader_step` 
+    - Breast Cancer Wisconsin: `sklearn_breast_cancer_loader_step`
+    - Wine: `sklearn_wine_loader_step`
```
Comment (fa9r) [inline README.md:19]: OpenML still missing here

File: src/tensorflow_dataset_steps/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Tensorflow Datasets Steps
+These are steps for loading small NumPy datasets from
```
Comment (fa9r) [inline src/tensorflow_dataset_steps/README.md:2]: Not sure about "small NumPy" datasets, e.g., FashionMNIST is 60K images

Comment (fa9r) [reply src/tensorflow_dataset_steps/README.md:2]: Ah lol just noticed "small NumPy datasets" is the wording in the Keras API docs as well, nevermind then 😆

**Review by fa9r — APPROVED** [review]
Awesome, love it, thanks so much!!!

File: README.md (hunk 1 above)
Comment (strickvl) [reply README.md:19]: Also, note to self, update the config file... https://github.com/zenml-io/zenml/commit/32ca90333ee12c3f347ca5818cad5d24fcf357a3

**Pull request merged**